test/compilable/ddocunittest.d

```diff
@@ -345,6 +345,8 @@ unittest { foo9757(); bar9757(); }
 ///
 unittest { bar9757(); foo9757(); }
 
+import std.functional : binaryFun;
```

Comment:
Phobos should ideally not be used in the tests.

Reply:
Oh oops. Hm.

Reply:
Fixed?